pyproject.toml

```
@@ -12,7 +12,7 @@ license = "Apache 2.0"
 [tool.poetry.dependencies]
 python = "<3.12,>=3.8.1"
 requests = "^2.25.1"
-singer-sdk = "^0.19.0"
+singer-sdk = "^0.20.0"
```

We added a set of "testing" extras in the SDK as part of the test framework release. This just includes `pytest` and `pytest-durations` right now, but it is safer to let the SDK manage the `pytest` version with the new tests 🙂

```suggestion
singer-sdk = {version = "^0.20.0", extras = ["testing"]}
```

---

This in addition to removing `pytest` in the dev dependencies on [L19](https://github.com/MeltanoLabs/tap-postgres/pull/51/files#diff-50c86b7ed8ac2cf95bd48334961bf0530cdc77b5a56f852c5c61b89d735fd711R19)

---

Done!